src/CommandAndStateMessageParser.cpp

```diff
@@ -160,6 +160,8 @@ string CommandAndStateMessageParser::parseTiltCameraHeadCommandMessage(string ap
     camera_head["lasers"] = head.laser;
     camera_head["tilt"]["speed"] =
         min(max(static_cast<double>(tilt.elements[0].speed), -1.0), 1.0) * 100;
+    m_camera_head_tilt_position_command =  min(max(static_cast<double>(tilt.elements[0].position), -110.0), 140.0);
```

Comment:
Let's repeat after me:

"All angles are represented using Radians. Conversion to degrees should be done only at the edge (i.e. just before presenting to the UI or sending to another API/Protocol that require it)"

In addition to being wrong per-se, base::samples::Joints will be representing angular positions as radians.